docs/developer/release/release_build.py

```
@@ -164,7 +164,7 @@ def get_current_date() -> str:
     Returns:
         today's date.
     """
-    return datetime.datetime.now().date().isoformat()  # noqa: DTZ005
+    return datetime.datetime.now().date().isoformat()  # ruff:ignore[DTZ005]
```

Ruff (0.15.21)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)